app/controllers/api/services_controller.rb

```diff
@@ -100,14 +101,30 @@ def service_params
   end
 
   def proxy_params
-    params.require(:service).fetch(:proxy_attributes, {}).permit(:sandbox_endpoint, :endpoint)
+    oidc_params = %i[oidc_issuer_type oidc_issuer_endpoint jwt_claim_with_client_id jwt_claim_with_client_id_type] + OIDCConfiguration::Config::FLOWS
+    permitted_params = oidc_params + %i[
+      auth_user_key auth_app_id auth_app_key credentials_location hostname_rewrite secret_token
+      error_status_auth_failed error_headers_auth_failed error_auth_failed
+      error_status_auth_missing error_headers_auth_missing error_auth_missing
+      error_status_no_match error_headers_no_match error_no_match
+      error_status_limits_exceeded error_headers_limits_exceeded error_limits_exceeded
+    ]
+    permitted_params << :api_backend unless apiap?
+    permitted_params += %i[sandbox_endpoint endpoint] if onpremises || service.deployment_option != 'hosted'
```

Comment:
Doh, we missed this
I think it should you should use `ThreeScale.config.apicast_custom_url`
See https://github.com/3scale/porta/blob/cc50d85499242cd9d404f67791f16d36d4f2e005/app/controllers/api/integrations_controller.rb#L237-L240

@macejmic correct me if I am mistaken please :)